src/test/java/hw3/po/Browser.java

```diff
@@ -0,0 +1,32 @@
+package hw3.po;
+
+import org.openqa.selenium.WebDriver;
+
+public class Browser {
```

Comment:
What are you trying realize in this class?

Reply:
I try to relize place to initialize and keep all pages.